sync_telegram/data/sync_project_data.xml

```diff
@@ -6,7 +6,6 @@
 <odoo>
     <record id="telegram_project" model="sync.project">
         <field name="name">Telegram Integration</field>
-        <field name="active" eval="False" />
```

Comment:
This still requires version change

Reply:
https://github.com/itpp-labs/sync-addons/pull/273